apps/webapp/app/v3/services/triggerTaskV1.server.ts

```
@@ -0,0 +1,758 @@
+                      queue: true,
+                    },
+                  },
+                },
+              },
+            },
+          })
+        : undefined;
+
+      if (
+        dependentBatchRun &&
+        dependentBatchRun.dependentTaskAttempt &&
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)